agent/php_wrapper.c

```diff
@@ -182,6 +186,11 @@ nruserfn_t* nr_php_wrap_generic_callable(zval* callable,
   zend_fcall_info_cache fcc;
   zend_fcall_info fci;
 
+  nr_wrap_user_function_options_t options = {
+    NR_WRAPREC_IS_TRANSIENT,
+    NR_WRAPREC_NO_INSTRUMENTED_FUNCTION_METRIC
+  };
```

Comment:
Could a note about these options be added to the doc string of this function? I.e.:
```
callable is wrapped with a transient wraprec and no instrumented function metric is being created for callables
```

Reply:
Added in [449afaa](https://github.com/newrelic/newrelic-php-agent/pull/730/commits/449afaacda9a9d734919944b3742d90d348eb3d9)